### DISCUSSION TITLE
What are the negative consequences using min_spiffs partition table?

### DISCUSSION THREAD

**Discussion by AndrewLivesChaos** [espressif/arduino-esp32#7425] [Q&A]
I'm wondering why spifss should have a bigger space than bare minimum. Is min_spiffs configuration a good choice for all kind of applications? Maybe my question is trivial but I don't know in the deep how a file system works. Thanks in advance.

**Comment by mrengineer7777** [comment]
https://github.com/espressif/arduino-esp32/tree/master/tools/partitions
It depends on what you want to do with your app.  min_spiffs will give you less space for file storage but more space for your app.

ESP32 modules range from 4MB to 32MB of flash.  Cost difference is minimal.  Consider buying one with more storage if you're low on space.  If OTA is not important consider using partition structure "no_ota.csv".

Comparison of some partition definitions:
![image](https://user-images.githubusercontent.com/44048235/199998979-29b75ce2-d8c7-44b7-ac2e-cbc07d5bc2a3.png)
